### ISSUE TITLE
Ability to enhance SSR ?

### ISSUE THREAD

**Issue by soupjake** [praydog/REFramework#693]
Hi was just wondering if we could use this to improve the engine's implementation of SSR? It's pretty low res and generally bad haloing for any RE title.  

Thanks!

**Comment by praydog** [comment]
Yes. I have played around with the resolution of SSR before, but I don't recall exactly in which class it's located in.

**Comment by soupjake** [comment]
via.render.SSRControl
via.render.SSRControl.SSRControlType
via.render.SSRControl.SSRResolvePointNum

I'm guessing one of these? Will have a mess around but I'm at a night time part of the game atm so hard to see reflections. Will play more til I get another good example of water.

**Comment by JoHien** [comment]
> via.render.SSRControl via.render.SSRControl.SSRControlType via.render.SSRControl.SSRResolvePointNum
> 
> I'm guessing one of these? Will have a mess around but I'm at a night time part of the game atm so hard to see reflections. Will play more til I get another good example of water.

Let us know how it goes.

**Comment by soupjake** [comment]
Seems like they're using some form of FidelityFX based SSR as all of the get calls in SSRControl return nil and I can see types for FidelityFXSSR. Just makes it a bit more complicated.

**Comment by JoHien** [comment]
> Seems like they're using some form of FidelityFX based SSR as all of the get calls in SSRControl return nil and I can see types for FidelityFXSSR. Just makes it a bit more complicated.

Dang, that's disappointing.
